selfdrive/controls/controlsd.py

```diff
@@ -723,6 +723,7 @@ def publish_logs(self, CS, start_time, CC, lac_log):
       self.pm.send('sendcan', can_list_to_can_capnp(can_sends, msgtype='sendcan', valid=CS.canValid))
       CC.actuatorsOutput = self.last_actuators
       self.steer_limited = abs(CC.actuators.steer - CC.actuatorsOutput.steer) > 1e-2
+      ControlsTimer.tick()
```

Comment:
What's the purpose of this?

Reply:
This increments the self.frame used in all carcontroller.py 's. I will separate this out to another PR later.